go/private/BUILD

```diff
@@ -2,7 +2,11 @@ load("@io_bazel_rules_go//go/private:go_toolchain.bzl", "go_toolchain_flags")
 
 filegroup(
     name = "all_rules",
-    srcs = glob(["*.bzl"]) + ["//go/private/tools:all_rules"],
+    srcs = glob(["*.bzl"]) + [
```

Comment:
Might be cleaner to just have a single BUILD file in go/private/BUILD. Then you could `glob(["**/*.bzl"])`. It doesn't seem like there's any need for these directories to be separate Bazel packages.

Reply:
Done